algorithms/FindersKeepers.js

```diff
@@ -0,0 +1,6 @@
+// Create a function that looks through an array (first argument) and returns the first element in the array that passes a truth test (second argument).
+
+function findElement(arr, func) {
```

Comment:
As a code quality practice, I try to avoid abbreviations for variable names, in order to optimize for readability by myself and other engineers.